diff --git a/node_modules/@rollup/plugin-commonjs/dist/es/index.js b/node_modules/@rollup/plugin-commonjs/dist/es/index.js
index 69f504a..307bab1 100644
--- a/node_modules/@rollup/plugin-commonjs/dist/es/index.js
+++ b/node_modules/@rollup/plugin-commonjs/dist/es/index.js
@@ -1124,7 +1124,8 @@ function getExportsForReplacedModuleExports(
 }
 
 function getDefaultExportDeclaration(exportedExportsName, defaultIsModuleExports, HELPERS_NAME) {
-  return `export default ${
+  return `export const isCommonJS = true;
+export default ${
     defaultIsModuleExports === true
       ? exportedExportsName
       : defaultIsModuleExports === false
